solutions/CSES Salary Queries.mdx

```diff
@@ -58,312 +56,28 @@ int main() {
 		cin >> ar[i];
 	}
 	vals = ar;
-    vector<array<int, 3>> rec;
-   	for (int i = 0; i < q; i++) {
-        char t;
+	vector<array<int, 3>> rec;
+	for (int i = 0; i < q; i++) {
+		char t;
 		int a,b;
 		cin >> t >> a >> b;
-        rec.push_back({t == '?', a, b});
-        if (t == '!') vals.push_back(b);
-    }
+		rec.push_back({t == '?', a, b});
+		if (t == '!') vals.push_back(b);
+	}
 	sort(vals.begin(), vals.end());
 	vals.erase(unique(vals.begin(), vals.end()), vals.end());
 	for (int i = 0; i < n; i++) {
 		ad(ar[i], 1);
 	}
-    for (auto u : rec) {
-        u[1]--;
-        if (u[0] == 0) {
-            	ad(ar[u[1]], -1);
-            	ar[u[1]] = u[2];
-            	ad(ar[u[1]], 1);
-        } else {
+	for (auto u : rec) {
+		u[1]--;
+		if (u[0] == 0) {
+				ad(ar[u[1]], -1);
+				ar[u[1]] = u[2];
+				ad(ar[u[1]], 1);
```

Comment:
extra tabs?

Reply:
ok now (before this commit, was mixture of tabs / spaces)